astropy/time/core.py

```diff
@@ -124,6 +124,16 @@ class Time(object):
     formats (e.g. JD) where very high precision (better than 64-bit precision)
     is required.
 
+    The allowed values for ``format`` and ``scale`` can be listed with::
+
+      >>> list(Time.FORMATS)
+      ['jd', 'mjd', 'decimalyear', 'unix', 'cxcsec', 'gps', 'plot_date', 'astropy_time',
+       'datetime', 'iso', 'isot', 'yday', 'fits', 'byear', 'jyear', 'byear_str',
+       'jyear_str']
+
+      >>> list(Time.SCALES)
+      ['tai', 'tcb', 'tcg', 'tdb', 'tt', 'ut1', 'utc']
```

Comment:
> How about not as much showing how to get them, but rather just inserting the values? 

The available formats are dynamic since user-supplied classes can be added.  That's why I wanted to show *how* to list the currently available formats instead of the built-in formats (available at the time of import).  The available time scales are (in practice) limited to the built-in scales, so that could be static.

:+1: to making `TIME_DELTA_SCALES` into an ordered dict.